tests/transmissionTest.js

```diff
@@ -0,0 +1,24 @@
+const { expect } = require("chai");
+const {
+  transmitInCelsius,
+  transmissionFailureCount,
+} = require("../transmitter.js");
+// Stub function for network transmission
+function networkTransmitStubSuccess(celsius) {
+  console.log(`Temperature  transmitted: ${celsius} Celsius`);
+  return 200;
+}
+function networkTransmitStubFailure(celsius) {
+  console.log(`Cannot transmit temperature : ${celsius} Celsius`);
+  return 500;
+}
+function transmissonTest() {
+  // Injecting the stub function when calling transmitInCelsius
+  transmitInCelsius(400.5, networkTransmitStubSuccess);
+  transmitInCelsius(303.6, networkTransmitStubSuccess);
+  // transmitInCelsius(504.7, networkTransmitStubFailure);
```

Comment:
Don't have commented code in commits. Remove it. If you need the older code, it will be there in GitHub